### DISCUSSION TITLE
Comment marche Galaxy Brain

### DISCUSSION THREAD

**Discussion by ARKANYOTA** [ARKANYOTA/ludumdare49#10] [Q&A]
Comment marche Galaxy Brain?

**Comment by ARKANYOTA** [comment]
Comme ça!

**Comment by ARKANYOTA** [comment]
Mais, il faut le faire 2 fois.

**Comment by Danilomd** [comment]
Achievements?

**Reply by ARKANYOTA** [reply]
I needed a repo for test if I could do the "galaxy brain" achievements, and what it took to do it, so I sent test messages here.
